core/tests/mod.rs

```diff
@@ -0,0 +1,646 @@
+use std::{
+    collections::{HashMap, HashSet},
+    path::Path,
+    str::FromStr,
+    sync::{Arc, Mutex},
+    time::Instant,
+};
+
+use iroha_config::iroha::Configuration;
+use iroha_core::{
+    block::BlockBuilder,
+    kura::Kura,
+    prelude::*,
+    queue::Queue,
+    smartcontracts::Registrable,
+    sumeragi::{message::*, view_change::*, *},
+};
+use iroha_data_model::{block::*, prelude::*, transaction::Accept};
+use iroha_genesis::{GenesisNetwork, GenesisNetworkTrait, RawGenesisBlock};
+use once_cell::sync::Lazy;
+use rand::{seq::IteratorRandom, SeedableRng};
+
+#[test]
+fn sumeragi_commits_and_broadcasts_genesis() {
+    init_telemetry();
+
+    let pb_hash = Arc::new(Mutex::new(None));
+    let pb_hash2 = pb_hash.clone();
+    /* Can't have 1 peer, then block is not sent. Can't have < 4 peers because non genesis peers get upset. */
```

Comment:
```suggestion
    // Can't have 1 peer, then block is not sent. Can't have < 4 peers because non genesis peers get upset
```

This form for me looks more modern and convinient with the rest of our code

Reply:
As you can tell, following a different convention with `//` is not a terrible idea.